crates/gosub_web_platform/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
```

Comment:
what does this to / what is it for?

Reply:
This will implement the "Web Platform", so things that a js or lua script can interact with. This might be timers, and Web APIs